apps/allocations/app/utils/constants.js

```diff
@@ -2,3 +2,9 @@ import { BigNumber } from 'bignumber.js'
 
 export const ETH_DECIMALS = new BigNumber(10e17)
 export const MIN_AMOUNT = new BigNumber(1e-18)
+export const STATUSES = {
+  0: 'Undergoing vote',
```

Comment:
An Enum like structure, would seem better than an object for this case, what are your thoughts on this?

Reply:
Definitely. Please feel free to update this.